api/client/inventory.go

```diff
@@ -514,7 +513,7 @@ func (i *upstreamICS) runSendLoop(stream proto.AuthService_InventoryControlStrea
 				sendMsg.errC <- trace.BadParameter("cannot send unexpected upstream msg type: %T", msg)
 				continue
 			}
-			err := trail.FromGRPC(stream.Send(&oneOf))
+			var err error = trace.Wrap(stream.Send(&oneOf))
```

Comment:
This should still work:

```suggestion
			err = trace.Wrap(stream.Send(&oneOf))
```